packages/rs-dapi-client/src/address_list.rs

```diff
@@ -4,22 +4,20 @@ use chrono::Utc;
 use dapi_grpc::tonic::codegen::http;
 use dapi_grpc::tonic::transport::Uri;
 use rand::{rngs::SmallRng, seq::IteratorRandom, SeedableRng};
-use std::collections::HashSet;
+use std::collections::hash_map::Entry;
+use std::collections::HashMap;
 use std::hash::{Hash, Hasher};
+use std::mem;
 use std::str::FromStr;
+use std::sync::{Arc, RwLock};
 use std::time::Duration;
 
 const DEFAULT_BASE_BAN_PERIOD: Duration = Duration::from_secs(60);
 
 /// DAPI address.
 #[derive(Debug, Clone, Eq)]
 #[cfg_attr(feature = "mocks", derive(serde::Serialize, serde::Deserialize))]
-pub struct Address {
-    ban_count: usize,
-    banned_until: Option<chrono::DateTime<Utc>>,
-    #[cfg_attr(feature = "mocks", serde(with = "http_serde::uri"))]
-    uri: Uri,
-}
+pub struct Address(#[cfg_attr(feature = "mocks", serde(with = "http_serde::uri"))] Uri);
```

Comment:
Do we still need the Address struct? Maybe we can just `type Address = Uri` ?

Reply:
Better to have newtype so we can implement traits we need and extend in future without breaking changes